### ISSUE TITLE
options prop only accept array of string

### ISSUE THREAD

**Issue by keweezy** [Codebrahma/React-combobox#1]
No way to pass an object to options prop. No way to get the id of a selected option

**Comment by TheRmbomo** [comment]
What's your code? I was able to pass in an object successfully. I just had to change the render options to handle that object, since React can't render objects by themselves.

`const options = [
  {id:'country-america',string:'America',toString(){return this.string}},
];
return (
  <ComboBox
    options={options}
    renderOptions={option=> <div className="comboBoxOption">{option.string}</div>}
    onSelect={option=> console.log(option.id)}
    onOptionsChange={option=> console.log(option, option.string)}
  />
);`

The only thing that isn't ideal at the moment is the input box only ever tries to display the current option itself, with no render option for it. A workaround for this is including a toString method within your option object, though this can only ever return a string. I even tried returning a <strong/>, which failed with 'TypeError: Cannot convert object to primitive value'.
